Allow OpenGL precision qualifiers for Qt5

From: Christian Prochaska <christian.prochaska@genode-labs.com>

Qt5 creates shader programs which use precision qualifiers. When these
programs get compiled for desktop OpenGL, a compile error occurs, because
a version number is too low. Lowering the version requirement avoids
the compile errors and didn't seem to cause any problems so far.
---
 src/mesa/shader/slang/slang_compile.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- src/lib/mesa/src/mesa/shader/slang/slang_compile.c
--- src/lib/mesa/src/mesa/shader/slang/slang_compile.c
@@ -2571,7 +2571,7 @@ parse_code_unit(slang_parse_ctx * C, slang_code_unit * unit,
 #if FEATURE_es2_glsl
    o.allow_precision = GL_TRUE;
 #else
-   o.allow_precision = (C->version >= 120) ? GL_TRUE : GL_FALSE;
+   o.allow_precision = (C->version >= 110) ? GL_TRUE : GL_FALSE;
 #endif
    init_default_precision(&o, unit->type);
 
